lib/graphql-hive/client.rb

```diff
@@ -24,6 +24,10 @@ def send(path, body, _log_type)
         request = build_request(uri, body)
         response = http.request(request)
 
+        if response.code.to_i != 200
```

Comment:
Small change: we only care about 400s or above (client or server errors).
```suggestion
        if response.code.to_i >= 400
```

Maybe `>= 400 && < 500` would also be okay, since we only want to see client-level errors, which means that we did something wrong, and not Hive.

Reply:
thanks for taking a look at this! updated here https://github.com/rperryng/graphql-ruby-hive/pull/44/commits/b75663bba3b67bd93dacc5b30bb49eb844af2512